### DISCUSSION TITLE
opensearch-operator-controller-manager not generating logs

### DISCUSSION THREAD

**Discussion by rameshar16** [Opster/opensearch-k8s-operator#917] [General]
Hi OpenSearch Community,

I am currently using the OpenSearch Operator, and I've encountered an issue where the opensearch-operator-controller-manager is not generating any new logs, even after setting the log level to debug. The last log entries I see are as follows:

`For more information, please go to https://github.com/brancz/kube-rbac-proxy/issues/187

===============================================

I1219 05:39:54.536474       1 kube-rbac-proxy.go:284] Valid token audiences:
I1219 05:39:54.536521       1 kube-rbac-proxy.go:378] Generating self signed cert as no cert is provided
I1219 05:40:34.938728       1 kube-rbac-proxy.go:442] Starting TCP socket on 0.0.0.0:8443
I1219 05:40:34.938975       1 kube-rbac-proxy.go:449] Listening securely on 0.0.0.0:8443
I1219 05:40:34.939007       1 kube-rbac-proxy.go:490] Starting TCP socket on 0.0.0.0:10443
I1219 05:40:34.939083       1 kube-rbac-proxy.go:497] Listening securely on 0.0.0.0:10443 for proxy endpoints`

After these entries, no further logs are being generated. Has anyone else experienced this issue or have any suggestions on how to resolve it?

Thanks in advance for your help!

**Comment by swoehrl-mw** [comment]
Hi @rameshar16. You are viewing the logs of the kube-rbac-proxy sidecar. To view logs for the main operator container use a command like `kubectl logs opensearch-operator-controller-manager-xxx -c operator-controller-manager`. There you should see regular log entries.

**Reply by rameshar16** [reply]
Thank you @swoehrl-mw. I can see the logs.
